Falied to retrieve git history for files in Docusarus build

<!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!
-->

**What happened**:

When building the Docusaurus app we get many error messages about missing git history:

```
#22 157.8 [ERROR] Error: Failed to retrieve the git history for file "/docs/i18n/uk/docusaurus-plugin-content-docs/current/overview.md" with exit code 128: fatal: not a git repository (or any of the parent directories): .git
#22 157.8 
#22 157.9 [ERROR] Error: Failed to retrieve the git history for file "/docs/docs/architecture.md" with exit code 128: fatal: not a git repository (or any of the parent directories): .git
```
**What should have happened**:

No git history errors during the build

**How to reproduce it (as precise as possible)**:

- `pnpm services:start`

--- a/src/web/docusaurus/docusaurus.config.js
+++ b/src/web/docusaurus/docusaurus.config.js
@@ -25,8 +25,6 @@ const config = {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
           editUrl: 'https://github.com/Seneca-CDOT/telescope/tree/master/src/web/docusaurus/',
-          showLastUpdateAuthor: true,
-          showLastUpdateTime: true,
           routeBasePath: '/',
           remarkPlugins: [require('mdx-mermaid')],
         },

